custom-bot-server-code-sample/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+
+const port = process.argv[2];
+const app = express();
+
+app.use(bodyParser.json());
+app.listen(port, function(){
+    console.log("The server is running on port: " +  port);
+});
+
+app.post("/",(req,res) => {
+    console.log(req.body); //request body contains the user message and other data from Kommunicate. More here : https://docs.kommunicate.io/docs/bot-custom-integration#integrating-with-custom-bot-platform
+
```

Comment:
add this line also: 

"You can get user's query from the request object and send it to your bot for further processing.  When you get the response send it back to Kommunicate in  response object. the request timeout is 30 sec."